bin/src/trust-dns.rs

```diff
@@ -401,7 +401,7 @@ fn main() {
         info!("binding UDP to {:?}", udp_socket);
         let udp_socket = runtime
             .block_on(UdpSocket::bind(udp_socket))
-            .unwrap_or_else(|_| panic!("could not bind to udp: {}", udp_socket));
+            .unwrap_or_else(|err| panic!("could not bind to udp: {}: {}", udp_socket, err));
```

Comment:
Nit: the `: {}: {}` seems a little messy. Maybe `could not bind to UDP socket {udp_socket}: {err}`? (We can switch to inline format arguments while we're at it.)

Reply:
I'm not a fan of inline format arguments, as they haven't got decent code insight in Rust Analyzer yet.
I will adjust though. In the future it will probably get better :)